### ISSUE TITLE
Strange translations in UK English

### ISSUE THREAD

**Issue by userj** [ubports/ubuntu-touch#907]
### Illustrations

- Device: Hammerhead
- Channel: Devel
- Build: 2018-10-10/2

### Steps to reproduce

- Change language to UK English
- Open Calendar or some settings with a "delete" dialogue

### Expected behavior

- Screen says "Delete"

### Actual behavior

- Screen says "Supprimer"

Sorry, I couldn't find this translation in translate.ubports.com. 
![screenshot20181001_191805005](https://user-images.githubusercontent.com/3128635/46741778-91cc2c00-cc5a-11e8-813c-126890b19ec6.png)

**Comment by dobey** [comment]
That's in the French translation. Wonder how it managed to pull that string rather than the en strings.

**Comment by Danfro** [comment]
BQ E5 16.04 rc 2018-W41 set language to english
Seems to effect other places as well.
At photo roll Editer and Supprimer sound very french as well...

In the German translation both words are translated correctly. It seems to affect english only!?

![screenshot20181010_220103106](https://user-images.githubusercontent.com/37637312/46762683-16a95b00-ccd8-11e8-81eb-fc5e3faec102.png)

![screenshot20181010_215018118](https://user-images.githubusercontent.com/37637312/46762164-b239cc00-ccd6-11e8-8091-b520ae1509eb.png)

**Comment by Flohack74** [comment]
Please try to check on https://translate.ubports.com

**Comment by myii** [comment]
@Danfro Found this issue since I've been having the same problems.  Had a look myself and found that some of the fixes have already been made:

* "Editer" in photo roll [changed to "Edit"](https://translate.ubports.com/translate/ubports/camera-app/en_GB/?checksum=5db700aa3ecc16d4#history)
* "Supprimer" in photo roll [changed to "Delete"](https://translate.ubports.com/translate/ubports/camera-app/en_GB/?checksum=d4a87b46e804e420#history)

What I would like to know is if there is any way of modifying these on my installation while waiting for the upstream changes to be released.

**Comment by myii** [comment]
Documentation pointed me `clickable` and I managed to update the calendar (and tested with another app successfully).  However, the camera app doesn't appear to be updated for `clickable` so couldn't do anything with that.

Apologies if stating the obvious, it's my first day with Ubports installed.

**Comment by userj** [comment]
@myii Thanks for updating the translations

**Comment by myii** [comment]
@userj Actually, the translation have already been done -- I was just linking to the commits when they've been changed since @Danfro didn't find them with the search because they'd already been fixed.

As for updating the apps, I only managed that on my personal installation.  Basically, all of the translations seem to have been fixed but we're waiting for the next release to receive them.

**Comment by userj** [comment]
@myii  Thanks for linking ;)

**Comment by UniversalSuperBox** [comment]
Is this still occurring in the latest version of the Gallery app? Looks like the po-files do not have these strings in them any more, it's possible it was all waiting on a rebuild.

**Comment by UniversalSuperBox** [comment]
It sounds like the *original issue* here is fixed, and these are different but related issues. To prevent this thread from becoming too cluttered, can you file a new issue with the currently incorrect translations at https://gitlab.com/ubports/app-dev/gallery-app/issues?
